src/VisualStudio/Core/Def/LanguageService/AbstractLanguageService`2.VsCodeWindowManager.cs

```diff
@@ -224,6 +232,66 @@ public int RemoveAdornments()
 
                 return VSConstants.S_OK;
             }
+
+            // We expect GetOutline to be called every time a new code window is created.
```

Comment:
Just to confirm it's not called if you are switching back and forth between two windows? The shell remembers the HWND itself?